src/app/design/guidelines/error-handling/index.html

```diff
@@ -27,6 +27,104 @@
 	<p>
 		Don't rely on a single sensory characteristic of components such as shape, size, visual location, orientation, or sound to describe an error or offer a solution. "Above" and "below" are fine for referencing content in the appropriate reading order for the language used as long as the reference isn't ambiguous.
 	</p>
+
```

Comment:
The accessibility guidelines at https://docs.blackbaud.com/skyua-patterns-docs/styletone/error-messages#accessibility-guidelines include "Error messages should be as specific as possible to help the user determine what is wrong." Should we add that here too?